.gitignore

```diff
@@ -30,7 +30,7 @@ doc/api/
 .idea/
 
 # Visual Studio Code related
-.vscode/
+# .vscode/
```

Comment:
why did you change that? .vscode (or any other editor specific files) should not be part of the code repository.

Reply:
Ok, I used my project settings (autoformat code for 120 length). I'll reformat to 80 length and remove this file